src/views/Game/PlayControls.tsx

```diff
@@ -749,6 +750,17 @@ export function AnalyzeButtonBar({
                     <i className="ogs-label-x"></i>
                 </button>
             </div>
+            {(!is_review || null) && (
+                <div className="btn-group">
+                    <button
+                        onClick={() => automateBranch(goban)}
+                        title={_("Copy branch to conditional move planner")}
+                        disabled={goban.engine.rengo}
```

Comment:
Should we add `data.get("user").id === currentPlayer(goban)` in this disabled statement too so the button is disabled if it's your move to make, or does that cause problems as we progress through the moves?

Reply:
There shouldn't be any problems. Also, maybe we should hide the button from rengo players since the button doesn't ever get enabled in rengo games?